README.md

```diff
@@ -174,9 +174,8 @@ binary package installation unless you want to join development and submit chang
    ```
    cd $COLCON_WS
    git clone https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver.git src/Universal_Robots_ROS2_Driver
```

Comment:
```suggestion
   git clone -b galactic https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver.git src/Universal_Robots_ROS2_Driver
```

We probably want to specify the correct version for this as well, just as in the foxy change